### PR TITLE
Update Indonesian translation

### PR DESCRIPTION
**Pull request by kodebanget**
A few contributors worked on this latest translation update in Crowdin: https://crowdin.com/project/bluesky-id/

### CODE REVIEW

File: src/locale/locales/id/messages.po [hunk 1]
```diff
@@ -1,76 +1,47 @@
...
 
 #: src/view/shell/Drawer.tsx:96
 msgid "<0>{0}</0> following"
-msgstr ""
+msgstr "<0>{0}</0> mengikuti"
 
 #: src/components/ProfileHoverCard/index.web.tsx:437
 msgid "<0>{followers} </0><1>{pluralizedFollowers}</1>"
-msgstr ""
+msgstr "<0>{followers} </0><1>{pluralizedFollowers}</1>"
```
Comment (kodebanget) [inline src/locale/locales/id/messages.po:44]: @pfrazee can you confirm, should this string be left as is or should it be replaced with the translation of "followers"?

Comment (pfrazee) [reply src/locale/locales/id/messages.po:44]: Oh that one, leave it as-is. It's not a useful translation string -- we'll get it fixed

**Review by danninov — COMMENTED** [review]
LGTM

**Review by pfrazee — APPROVED** [review]

**Comment by pfrazee** [comment]
Thanks everyone!!

**Pull request merged**